docs/whats-new.rst

```diff
@@ -8,6 +8,13 @@ What's New
 v0.2.1 (unreleased)
 -------------------
 
+Enhancements
+~~~~~~~~~~~~
+
+- Remove ``xarray.Dataset`` attributes from example netcdf files.
```

Comment:
Strictly speaking, these are attributes of the netCDF files themselves, in addition to the `xr.Dataset` representations of the files.  So replace "``xarray.Dataset``" attributes with "potentially confusing".